### ISSUE TITLE
2.1.6 Release Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1414]
- #1417
- #1422

**Comment by joeparsons** [comment]
## Pick list

```
git cherry-pick c588c0e98cb922ed425626c7e2f27737bfd693fb
git cherry-pick bd1a7b8adc69254f3a5cc990612ab8a0e0f4b7df
```

**Comment by joeparsons** [comment]
## Release Notes Snippet

```
_This release contains a security update for Drupal core addressing a moderately critical security vulnerability ([SA-CORE-2022-006](https://www.drupal.org/sa-core-2022-006))._
```

`git log 2.2.2..HEAD --pretty='format:- %s'`
```
- Closes #1421: Update Drupal core to 9.2.16 for SA-CORE-2022-006. (#1422)
- Fixes #1415: Add required default_theme property to all tests. (#1417)
```

**Comment by joeparsons** [comment]
Released.
